x/ibc/09-localhost/types/msgs.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	clientexported "github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
+)
+
+// Message types for the IBC client
+const (
+	TypeMsgCreateClient string = "create_client"
+)
+
+var (
+	_ clientexported.MsgCreateClient = MsgCreateClient{}
+)
+
+// MsgCreateClient defines a message to create an IBC client
+type MsgCreateClient struct {
```

Comment:
Should every chain just create a `localhost` client by default and we can just remove this msg?

Reply:
mmmmm idk what's the benefit of creating one by default? @zmanian thoughts?

Reply:
It depends if there's any reason to want more than one localhost client - I really can't think why there would be, in which case it does seem prudent to create one by default.